queries.sql

```diff
@@ -5,4 +5,61 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--UPDATE SPECIES COLUMN
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+
+COMMIT;
+SELECT * FROM animals;
+
```

Comment:
- [ ] Before the transaction highlighted in the code snippet, you ought to write a transaction that sets all `species` column to `'unspecified'` and then roll it back. In the process of the transaction, you ought to verify that the columns were set to `'unspecified'` by selecting all from the `animals` table. Kindly carry out this transaction as it is a requirement. 👍 💻 

![image](https://github.com/otmaneechchafyky/vet_linic_database/assets/77106962/8c1f2117-3ff7-4637-b5bf-694de4091070)

### N:B: Please update the screenshots in the pull request summary with the verified changes of this transaction. 😎